src/libs/suomifi-utils.ts

```diff
@@ -0,0 +1,148 @@
+import { Service, SuomiFiResponseWithMeta, SupportedLanguages } from "@types";
+import fetch from "node-fetch";
+import config from "src/config";
+import { Department } from "./departments-registry-utils";
+
+const { SUOMIFI_HELSINKI_ORGANIZATION_ID } = config;
+const suomifiBasePath = "https://api.palvelutietovaranto.suomi.fi/api/v11/"
+const suomifiServicesPath = "Service/list/organization?organizationId=";
+
+/**
+ * Gets services from Suomi.fi API
+ */
+export const getSuomifiServices = async () => {
```

Comment:
What is the purpose of this mehtod? At least from first glance it seems that it might lead to huge number of requests, so would it be possible to reduce that amount by using the organization entity's service id list as basis?

Reply:
REsolved